packages/accordion/src/Accordion.ts

```diff
@@ -41,6 +43,9 @@ export class Accordion extends SpectrumElement {
     @property({ type: Boolean, reflect: true, attribute: 'allow-multiple' })
     public allowMultiple = false;
 
+    @property({ type: String, reflect: true })
+    public density?: 'compact' | 'regular' | 'spacious' = 'regular';
```

Comment:
```suggestion
    public density?: 'compact' | 'spacious';
```

We shouldn't need to manage an attribute for 'regular', since this density property is optional and "regular" is the default, anyway. Please remove this and its usages.

Reply:
If we remove regular from here, documentation may not have this option.

Reply:
That's okay. It's not needed. Please refer to some of [our](https://opensource.adobe.com/spectrum-web-components/components/action-button/) [other](https://opensource.adobe.com/spectrum-web-components/components/toast/) [APIs](https://opensource.adobe.com/spectrum-web-components/components/button/) to see that we don't document the "standard" option. Just make sure to fill out the JS Doc for this component so that it shows up in the API tab on the docs site.

While you're at it, please also add a segment to the documentation for the different densities, as those are new to the API.

Reply:
updated the examples and JS Doc string.